tests/test_password_input.py

```diff
@@ -0,0 +1,61 @@
+from vorta.views.partials.password_input import PasswordInput, PasswordLineEdit
+
+
+def test_create_password_line_edit(qtbot):
+    password_line_edit = PasswordLineEdit()
+    qtbot.addWidget(password_line_edit)
+    assert password_line_edit is not None
+
+
+def test_password_line_get_password(qtbot):
+    password_line_edit = PasswordLineEdit()
+    qtbot.addWidget(password_line_edit)
+
+    password_line_edit.setText("test")
+    assert password_line_edit.get_password() == "test"
+
+
+def test_password_line_visible(qtbot):
+    password_line_edit = PasswordLineEdit()
+    qtbot.addWidget(password_line_edit)
+    assert password_line_edit.visible is False
+
+    password_line_edit.toggle_visibility()
+    assert password_line_edit.visible
+
+
+def test_password_line_error_state(qtbot):
+    password_line_edit = PasswordLineEdit()
+    qtbot.addWidget(password_line_edit)
+    assert password_line_edit.error_state is False
+
+    password_line_edit.error_state = True
+    assert password_line_edit.error_state
+    assert password_line_edit.styleSheet() == "border: 2px solid red;"
+
+
+def test_password_lint_visibility_button(qtbot):
```

Comment:
`password_line_visibility_button`?

Reply:
The tests are good; coverage is at 91% and will be easy to increase.

Reply:
Thank you, fixed.